docs/source/monitor.rst

```diff
@@ -9,6 +9,16 @@ torch.monitor
 ``torch.monitor`` provides an interface for logging events and counters from
 PyTorch.
 
+The stat interfaces are designed to be used for tracking high level metrics that
+are periodically logged out to be used for monitoring system performance. Since
```

Comment:
"logged out" => "logged to external systems" or just "logged"?

~~to be used~~